ci/runner/Dockerfile

```diff
@@ -67,6 +67,7 @@ COPY ./docker/conda/environments/cuda${CUDA_SHORT_VER}_examples.yml /tmp/conda/c
 COPY ./ci/scripts/download_kafka.py /tmp/scripts/download_kafka.py
 
 # Install extra deps needed for gnn_fraud_detection_pipeline & ransomware_detection examples
+# can stellargraph be obtained through conda?
```

Comment:
Is this comment intentional?

Reply:
I saw we were using pip to install stellargraph here and left the comment so I wouldn't forget to check on it later. Guess it worked, in a round-about way. Should we try to install stellargraph via conda and get rid of this pip install call?

Reply:
Lets leave it alone for now. There is a PR: #1032 which will remove stellargraph entirely. No need to worry about it for now.